documentation/configuration.markdown

```diff
@@ -75,9 +79,9 @@ This file contains the settings for Puppet Server itself.
 # configuration for the JRuby interpreters
 
 jruby-puppet: {
-    gem-home: /var/lib/puppet/jruby-gems
+    gem-home: /opt/puppetlabs/puppet/cache/jruby-gems
```

Comment:
Ok.  The only point I wanted to make here is that I think it would be good to have the canonical "file_paths.md" also updated to whatever we set for this in puppet-server since that covers puppet-server paths, too.  Maybe that's already being taken care of independently?